book/2 - Configuration Basics/3 - WebAdmin.html

```diff
@@ -1,5 +1,6 @@
 <p>
 	The WebAdmin allows you to control various aspects of Cuberite, including player permissions.
+	By default, the webadmin is disabled. To enable it, you need to configure it properly.
```

Comment:
This isn't entirely true right? The webadmin is enabled by default, it just doesn't have any users.

Reply:
Indeed but from a simple user POV it looks like this.

Maybe add explanation that there can be multiple users

Reply:
I've created https://github.com/cuberite/cuberite/pull/5499 which should give a warning if the webadmin was enabled without any users.

Reply:
How about `By default, the webadmin is enabled but has no users. To use the webadmin it needs to be configured properly.`

Reply:
This one is great